skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -143,12 +134,13 @@ def report_metrics(
         Recall                 0.93...         (↗︎)
         ROC AUC                0.99...         (↗︎)
         Brier score            0.03...         (↘︎)
+        >>> # Using scikit-learn metrics
+        >>> report.metrics.report_metrics(scoring=["neg_log_loss"])
+                    LogisticRegression
+        Metric
+        Log Loss             -0.10...
```

Comment:
It should rather say something like
```suggestion
        neg_log_loss             -0.10...
```
or maybe
```suggestion
        Negative Log Loss             -0.10...
```